index.js

```diff
@@ -269,11 +269,35 @@ const publicApi = {
      * For example, if you want to add the "webpack.IgnorePlugin()", then:
      *      .addPlugin(new webpack.IgnorePlugin(requestRegExp, contextRegExp))
      *
+     * By default custom plugins are added after the ones managed by Encore
+     * but you can also set a priority to define where your plugin will be
+     * added in the generated Webpack config.
+     *
+     * For example, if a plugin has a priority of 0 and you want to add
+     * another plugin before it, then:
+     *
+     *      .addPlugin(new MyWebpackPlugin(), -10)
```

Comment:
We need to reverse the sorting: priority means that a *higher* number wins. So `-10` should run *after* plugins with priority 0. This is also consistent with Symfony's priorities.

Reply:
Whoops, fixed :)